components/ActivityCard.js

```diff
@@ -0,0 +1,66 @@
+import Link from "next/link";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function ActivityCard({ title, categories, imageUrl }) {
+  return (
+    <StyledCardSection>
+      <StyledImageSection>
+        <Image
+          src={imageUrl}
+          alt={title}
+          style={{ objectFit: "cover" }}
+          sizes="33vw"
+          fill
+        />
+      </StyledImageSection>
+      <StyledDetailsSection>
+        <StyledList>
+          {categories.map((category, index) => (
+            <StyledListItem key={index}>{category}</StyledListItem>
+          ))}
+        </StyledList>
+        <StyledLink href="#" title={title}>
+          {title}
+        </StyledLink>
+      </StyledDetailsSection>
+    </StyledCardSection>
```

Comment:
you generally don't put sections into sections. They are meant to divide the content of entire pages into distinct elements. Your card should be an `article` and I think you don't need the ImageSection and the DetailsSection, they can be removed to make the html easier. If you still need them use divs. Your Title should not be a link (yet). There is nothing to point to.